acceptancetests/README.md

```diff
@@ -1,3 +1,114 @@
+# Juju Functional Tests
+
+## Running Test Scripts 
+
+Some setup is required before running the tests.
+
+### Installing dependencies
+
+Dependencies are installed via a .deb and pip install.
+
+```bash
+$ sudo apt-get install equivs
+
+$ make install deps
```

Comment:
We actually ignore the Makefile to update our slaves because of dependencies issues -- we should sync those back. However, specifically for instance we user install to avoid some bad things `pip install --user -r requirements.txt`. This needs cleaned up however.